website/package.json

```diff
@@ -20,39 +20,39 @@
   "author": "Patronage.org",
   "license": "private",
   "dependencies": {
-    "@popperjs/core": "^2.9.3",
-    "bootstrap": "^5.1.0",
+    "@popperjs/core": "^2.10.2",
+    "bootstrap": "^5.1.3",
     "classnames": "^2.3.1",
-    "dayjs": "^1.10.6",
+    "dayjs": "^1.10.7",
     "isomorphic-unfetch": "^3.1.0",
-    "next": "^11.1.0",
-    "next-seo": "^4.26.0",
+    "next": "^12.0.4",
+    "next-seo": "^4.28.1",
     "react": "^17.0.2",
```

Comment:
next, react, and react-dom should be updated in tandem. Are react/react-dom both on latest version?

Reply:
I use this to update deps: https://www.npmjs.com/package/npm-check-updates

It's the latest version of everything except for eslint which has issues (see comment earlier in thread). The current version of react is 17.0.2.